src/App.jsx

```diff
@@ -1,10 +1,29 @@
-import './css/App.css'
+import "./css/App.css";
+import { Route, Routes } from "react-router-dom";
+import { useState } from "react";
+import Header from "./1.Header";
+import Home from "./2.Home";
+import Topics from "./3.Topics";
+import Articles from "./4.Articles";
+import Users from "./5.Users";
 
 function App() {
+  const [articles, setArticles] = useState([]);
```

Comment:
Could you create this state in the Articles component? Or do you plan on using it elsewhere? Currently only being passed to Articles